src/components/Search/Search.js

```diff
@@ -7,7 +7,7 @@ function Search(props) {
       <h1 className="search__title">{props.searchTitle}</h1>
       <p className="search__subtitle">{props.searchSubtitle}</p>
       <form name="query" id="search-form" className="search__form">
-          <input id="search-form-input" className="search__form-input" placeholder={props.searchPlaceholder}></input>
+          <input ref={props.inputRef} id="search-form-input" className="search__form-input" placeholder={props.searchPlaceholder}></input>
           <button className="search__form-button" onClick={props.onSearchSubmit}>{props.searchButtonText}</button>
```

Comment:
` onClick={props.onSearchSubmit}`

please, delete it from the button

Submit handlers are added only to tags `form` with event `submit` rather than to submit buttons with event `click` because it also automatically handles `Enter` presses